src/main/java/website/frontrow/board/Enemy.java

```diff
@@ -21,6 +21,7 @@
 	private float random = 1.00f;
 	private Boolean lastWall = false;
 	private Boolean wallCollision = false;
+	private Mover player;
```

Comment:
Why use Mover as type instead of Player?
Or in case you want the AI to target things other than just players:
Why did you name it player and not `target`?

< naming